components/MatchDetailsForm/index.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components"
+
+export default function MatchDetailsForm({bars, onSubmit, currentMatch}){
+
+const barsToAdd = bars.filter((bar) => (
+        !bar.matches.includes(currentMatch?.id)
+    ))
+
+    return(
+        <Form onSubmit={onSubmit}>
```

Comment:
Just an information: For better accessibility it would be nice to add an accessible name to this form. You could solve this problem by using `aria-labelledby` for your form element that relates to an id that you add to the `h4` element in your details page:

```jsx
 <Form onSubmit={onSubmit} aria-labelledby="match-details-form">
```

```jsx
       <h4 id="match-details-form">Du weißt wo es läuft?</h4>
        <MatchDetailsForm bars={bars} onSubmit={handleSubmit} currentMatch={currentMatch}/>
```

Feel free to change this if you like. :) [You can read more about it on MDN if you want to.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/form_role#description)